crdesigner/common/config/opendrive_config.py

```diff
@@ -12,7 +12,7 @@ class OpenDriveConfig(BaseConfig):
     )
 
     min_delta_s = Attribute(
-        0.5, "Min. delta s", "Min. step length between two sampling positions on the reference geometry"
+        1, "Min. delta s", "Min. step length between two sampling positions on the reference geometry"
```

Comment:
Instead of changing the default here, can you set in in `convert_map.py`? It will reduce the conflicts when merging upstream changes.